btr-web/btr-common-components/cypress/e2e/pages/examples/form/full-name.cy.ts

```diff
@@ -36,18 +36,18 @@ describe('forms -> preferred name -> validate that the preferred name component
 
   it('test the validation rule for special character', () => {
     cy.on('uncaught:exception', (err) => {
-      expect(err.message).to.include('The legal name should not contain special character')
+      expect(err.message).to.include('The legal name should not contain special characters')
       return false
     })
 
     const invalidName = 'first - last'
     const validName = 'first last'
 
     cy.get('#testFullName').type(invalidName).blur()
-    cy.contains('The legal name should not contain special character').should('exist')
```

Comment:
This is something I have been looking into. We should include lang JSON files and have `contains`  match those. Or at least this one should be a constant in the file.